maltd-frontend/src/components/composite/UserSearch/UserSearch.js

```diff
@@ -17,7 +17,7 @@ export default function UserSearch({
         projects.
       </p>
       <p />
-      {!state.isLoading && (
+      {!state.isLoading && state.userExists && (
```

Comment:
the trailing && is a little confusing, is this a react thing I'm not aware of? sorry for my ignorance

Reply:
yeah it is a way to add conditional statements in react, this is fine